src/fragments/lib/push-notifications/ios/getting_started/40_cli_resources.mdx

```diff
@@ -0,0 +1,47 @@
+> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
```

Comment:
can we actually change this section to be more generic around "setting up backend resources" - and then have a tab switcher between using the CLI VS using existing resources. All categories are going to start moving to that type of format soon - so might as well get ahead of that. The way - the journey isn't disconnected by having developers jump to another page and then come back to getting started for setting up the library.

Reply:
done.